packages/popover/src/vaadin-popover-overlay.js

```diff
@@ -62,8 +62,7 @@ class PopoverOverlay extends PopoverOverlayMixin(DirMixin(ThemableMixin(PolylitM
           content: '';
           top: calc(var(--vaadin-popover-offset-top, 0) * -1);
           bottom: calc(var(--vaadin-popover-offset-bottom, 0) * -1);
```

Comment:
Could this also be shortened with `inset-block`?

Reply:
Done.